app/src/pages/inside/uniqueErrorsPage/uniqueErrorsGrid/nestedGridRow/nestedGridRow.jsx

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { defineMessages, useIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+import Parser from 'html-react-parser';
+import { SpinningPreloader } from 'components/preloaders/spinningPreloader';
+import { StepGrid } from 'pages/inside/stepPage/stepGrid';
+import { StackTraceMessageBlock } from 'pages/inside/common/stackTraceMessageBlock';
+import ArrowIcon from 'common/img/arrow-right-inline.svg';
+import {
+  clusterItemsSelector,
+  loadMoreClusterItemsAction,
+  requestClusterItemsAction,
+  selectClusterItemsAction,
+  selectedClusterItemsSelector,
+  toggleAllClusterItemsAction,
+  toggleClusterItemSelectionAction,
+} from 'controllers/uniqueErrors/clusterItems';
+import { ENTITY_METHOD_TYPE } from 'components/filterEntities/constants';
+import { PROJECT_LOG_PAGE } from 'controllers/pages';
+import { reloadClusterAction } from 'controllers/uniqueErrors';
+import { modifyColumnsFunc } from './utils';
+import styles from './nestedGridRow.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  loadLabel: {
+    id: 'NestedGridRow.loadLabel',
+    defaultMessage: 'Load 10 more',
+  },
+});
+
+const ClusterColumn = ({ cluster }) => (
+  <StackTraceMessageBlock
+    level={'error'}
+    maxHeight={75}
+    customProps={{
+      rowWrapper: cx('cluster-row-wrapper'),
+      row: cx('cluster-row'),
+      accordionBlock: cx('cluster-accordion-block'),
+    }}
+  >
+    <div>{cluster.message}</div>
+  </StackTraceMessageBlock>
+);
+ClusterColumn.propTypes = {
+  cluster: PropTypes.object,
+};
+ClusterColumn.defaultProps = {
+  cluster: {},
+};
+
+export const NestedGridRow = ({ data, onEditItem, onUnlinkSingleTicket, onEditDefect }) => {
```

Comment:
Please call it as `ClusterItemsGridRow` or smth like that as this component represents the cluster items list.